css/includes/components/_richtext.scss

```
@@ -70,7 +70,8 @@ body.mce-content-body {
     }
 
     table {
-        word-break: normal;
+        overflow: auto;
+        max-width: 100%;
```

See #13038. We should probably preserve the `word-break: normal;` property.
```suggestion
        max-width: 100%;
        word-break: normal;
```